### PR TITLE
Close changelog for 7.3.1

### PR DESCRIPTION
**Pull request by mikemadden42**

### CODE REVIEW

**Comment by ycombinator** [comment]
@mikemadden42 Added missing CHANGELOG entries in https://github.com/elastic/beats/pull/13289 for #13133, #13020, and #13082.

**Comment by mikemadden42** [comment]
jenkins retest this please

**Comment by mikemadden42** [comment]
jenkins test this

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -3,6 +3,31 @@
...
+*Metricbeat*
+
+- Fix `logstash/node_stats` metricset to also collect `logstash_stats.events.duration_in_millis` field when `xpack.enabled: true` is set. {pull}13082[13082]
+- Fix `logstash/node` metricset to also collect `logstash_state.pipeline.representation.{type,version,hash}` fields when `xpack.enabled: true` is set. {pull}13133[13133]
+
+==== Added
+
+*Metricbeat*
+
+- Make the `beat` module defensive about determine ES cluster UUID when `xpack.enabled: true` is set. {pull}13020[13020]
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]:
```suggestion
- Make the `beat` module defensive about determining ES cluster UUID when `xpack.enabled: true` is set. {pull}13020[13020]
```

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -3,6 +3,31 @@
...
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix install-service.ps1's ability to set Windows service's delay start configuration. {pull}13173[13173]
+
+*Filebeat*
+
+- Fix multiline pattern in Postgres which was too permissive {issue}12078[12078] {pull}13069[13069]
```
Comment (dedemorton) [inline CHANGELOG.asciidoc]:
```suggestion
- Fix multiline pattern in Postgres which was too permissive. {issue}12078[12078] {pull}13069[13069]
```

**Comment by kvch** [comment]
Added missing entry for #13092 and #13144: https://github.com/elastic/beats/pull/13298

**Comment by elasticcla** [comment]
Hi @mikemadden42, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in your Git [commit](https://github.com/elastic/beats/pull/13277.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Pull request merged**